docs/controls/ImageCropper.md

```diff
@@ -0,0 +1,102 @@
+---
+title: ImageCropper
+author: HHChaos
+description: ImageCropper control allows user to crop image freely.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, ImageCropper
+dev_langs:
+  - csharp
+---
+
+# ImageCropper XAML Control
+
+The [ImageCropper Control](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.controls.imagecropper) allows user to crop image freely.
+
+## Syntax
+
+```xaml
+<Page ...
+     xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"/>
+    <controls:ImageCropper x:Name="ImageCropper" />
+</Page>
+```
+
+## Sample Output
+
+![ImageCropper animation](../resources/images/Controls/ImageCropper.gif)
+
+## Properties
+
+
+| Property              | Type            | Description                                                  |
+| --------------------- | --------------- | ------------------------------------------------------------ |
+| MinCroppedPixelLength | double          | Gets or sets the minimum cropped length(in pixel).           |
+| MinSelectedLength     | double          | Gets or sets the minimum selectable length.                  |
+| CroppedRegion         | Rect            | Gets the current cropped region.                             |
+| Source                | WriteableBitmap | Gets or sets the source of the cropped image.                |
+| AspectRatio           | double?         | Gets or sets the aspect ratio of the cropped image, the default value is null. |
+| CropShape             | CropShape       | Gets or sets the shape to use when cropping.                 |
+| Mask                  | Brush           | Gets or sets the mask on the cropped image.                  |
+| PrimaryThumbStyle     | Style           | Gets or sets a value for the style to use for the primary thumbs of the ImageCropper. |
+| SecondaryThumbStyle   | Style           | Gets or sets a value for the style to use for the secondary thumbs of the ImageCropper. |
+| ThumbPlacement        | ThumbPlacement  | Gets or sets a value for thumb placement.                    |
+
+
+## Methods
+
+| Methods                                              | Return Type | Description                                                  |
+| ---------------------------------------------------- | ----------- | ------------------------------------------------------------ |
+| LoadImageFromFile(StorageFile)                       | Task        | Load an image from a file.                                   |
+| SaveAsync(IRandomAccessStream,BitmapFileFormat,bool) | Task        | Saves the cropped image to a stream with the specified format. |
```

Comment:
```suggestion
| SaveAsync(IRandomAccessStream,BitmapFileFormat,bool) | Task        | Saves the cropped image to a stream with the specified format. Setting the boolean argument to True will save pixel values to the extent of the cropped area regardless of the crop shape, otherwise transparent or black pixels will fill the uncropped area depending on file format. |
```